widgets/WebComponents/AvatarIcon/AvatarIcon.bbj

```diff
@@ -0,0 +1,46 @@
+use ::WebKit/widgets/WebComponents/WebComponents.bbj::WebComponents
+use ::WebKit/util/ClientUtil.bbj::ClientUtil
+
+class public AvatarIcon extends WebComponents
+    field private static BBjTopLevelWindow Wnd!
+    field public static BBjNumber ON_AVATAR_INITIAL_CLICK = 1234
+
+    method public AvatarIcon()
+
+    methodend
+
+    method public BBjHtmlView drawAvatarIcon(BBjWindow wnd!,BBjInt height%, BBjInt width%, BBjString name$)
+        htmlView! = wnd!.addHtmlView(wnd!.getAvailableControlID(),0,0,300,300,"<avatar-initial height="""+str(height%)+""" width="""+str(width%)+""" name="""+name$+"""></avatar-initial>")
+        htmlView! = #configureAvatarView(htmlView!)
+        methodret htmlView!
+    methodend 
+
+    method public BBjHtmlView drawAvatarIcon(BBjWindow wnd!,BBjString height$, BBjString width$, BBjString name$)
+        htmlView! = wnd!.addHtmlView(wnd!.getAvailableControlID(),0,0,300,300,"<avatar-initial height="""+height$+""" width="""+width$+""" name="""+name$+"""></avatar-initial>")
+        htmlView! = #configureAvatarView(htmlView!)
+        methodret htmlView!
+    methodend 
+
+    method public BBjHtmlView drawAvatarIcon(BBjString height$, BBjString width$, BBjString name$)
```

Comment:
The class should be a widget meaning that the BBjHtmlView should not be returned. The class might use the BBjHtmlView internally but for external usage it should always be treated as BBjWidget